File: 백준/Bronze/1978. 소수 찾기/소수 찾기.cc
#include <iostream>

using namespace std;

int main() {

    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int N;
    cin >> N;
    int count = 0;

    for (int i = 0; i < N; i++) {
        int input = 0;
        cin >> input;

        if (input < 2) continue;
        bool isPrime = true;
        for (int j = 2; j < input; j++) {
            if (input % j == 0) {
                isPrime = false;
                break;
            }
        }
        
        if (isPrime) {
            count++;
        }
    }

    cout << count;

    return 0;
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
n = int(input())
num_list = list(map(int, input().split()))
count = 0

for num in num_list:
  if (num < 2): continue
  isPrime = True
  for i in range(2, num):
    if (num % i == 0):
      isPrime = False
      break
    
  if (isPrime): count += 1
print(count)